recipes/zeromq/all/zeromq_extra.cmake

```diff
@@ -0,0 +1,13 @@
+# This script should only run when included in a FindXXX.cmake module
```

Comment:
I really don't like these extras cmake files. I hope having https://github.com/conan-io/conan/issues/5090 soon.

Reply:
Yes, I will add a comment about eventually adding the missing targets/variables.